### DISCUSSION TITLE
Enabling "syntax: template-literal" does not generate patterns

### DISCUSSION THREAD

**Discussion by axelfriberg** [chakra-ui/panda#949] [General]
### Description

When enabling template-literal syntax in the config

```
import { defineConfig } from '@pandacss/dev';

export default defineConfig({
  preflight: true,
  include: ['./components/**/*.{ts,tsx,js,jsx}', './app/**/*.{ts,tsx,js,jsx}'],
  exclude: [],
  theme: {
    extend: {},
  },
  outdir: 'styled-system',
  jsxFramework: 'react',
  syntax: 'template-literal',
});
```

And then running `panda codegen --clean` the "patterns" folder is no longer generated. So that things like `container` can't be used anymore.

If this is a know limitation about the current setup around template-literals something could be added to the docs to explain this.

### Link to Reproduction

https://stackblitz.com/edit/stackblitz-starters-m7oq7c?file=panda.config.mjs

### Steps to reproduce

1. Set syntax to template-literal in config.
2. panda codegen --clean
3. No patterns are generated under styled-system.

### JS Framework

React (next.js 13)

### Panda CSS Version

0.5.1

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Hi @axelfriberg,

Yes, this is a limitation of the template literal syntax. We intend to write some docs around this shortly.

**Reply by axelfriberg** [reply]
All right, thanks for the info Segun!
